queue/queue.py

```diff
@@ -1,15 +1,23 @@
 class Queue:
```

Comment:
Great job. How could this be done with a Linked List instead of an array? (Not required for you to change it, just a thought)